bucket/todolist.json

```
@@ -41,8 +41,9 @@
         "Resources\\TaskLists"
     ],
     "checkver": {
-        "url": "https://github.com/abstractspoon/ToDoList_Downloads/tree/master/Latest",
-        "regex": "([\\d.]+)</a>"
+        "url": "https://api.github.com/repos/abstractspoon/ToDoList_Downloads/commits",
+        "jsonpath": "$..message",
+        "regex": "Update to ([\\w\\d.]+)"
```

```suggestion
        "regex": "Update to (\\d+\\.\\d+\\.[\\d.]+)"
```

We download `todolist_exe.zip` so avoid commit message for `todolist_exe_pre.zip`.